chain-impl-mockchain/tests/common/address.rs

```diff
@@ -24,6 +40,19 @@ impl AddressData {
         }
     }
 
+    pub fn as_input(&self, value: Value, utxo: UtxoPointer) -> Input {
```

Comment:
Conventionally, `as_` methods denote lightweight by-reference conversions.
This should be named `to_input` or something more meaningful to the subject matter, maybe `make_input`?